elephant/sta.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+'''
+Function to calculate spike-triggered averages of analog signals.
+
+:copyright: Copyright 2015 by the Elephant team, see AUTHORS.txt.
+:license: Modified BSD, see LICENSE.txt for details.
+'''
+
+import numpy as np
+import quantities as pq
+from neo.core import AnalogSignalArray, SpikeTrain
+
+
+def spike_triggered_average(signal, spiketrains, window):
+    """
+    Calculates the spike-triggered average of analog signals in a given time window around 
+    the spike times of a corresponding spiketrain for multiple signals each.
+
+    The function receives n analog signals and either one or n spiketrains. In case it is one spiketrain 
+    this one is muliplied n-fold and used for each of the n analog signals.
+
+    Parameters
+    ----------
+
+        signal: neo AnalogSignalArray object containing n analog signals
```

Comment:
This description style of parameters is not following NumPy documentation guideline.
I would suggest to revise it to comply with the guideline.